vendor/src/github.com/docker/engine-api/client/image_build.go

```diff
@@ -79,6 +79,7 @@ func imageBuildOptionsToQuery(options types.ImageBuildOptions) (url.Values, erro
 	}
 
 	query.Set("cpusetcpus", options.CPUSetCPUs)
+	query.Set("netmode", options.NetMode)
```

Comment:
these changes go in https://github.com/docker/engine-api. Then, we need to revendor the project.